rkt/view-activities.rkt

```diff
@@ -350,6 +350,17 @@ select X.session_id
                   fn
                   #:default-visible? #t))
 
+       (let ((fn (lambda (row)
+                   (let* ((length (db-row-ref row "pl" headers 0))
+                          (unit (db-row-ref row "plu" headers 0))
+                          (pool-length (if (equal? unit 0)
+                                           length
+                                           (*  length 0.9144))))
+                     (if (equal? pool-length 0)
+                         "" 
+                         (short-distance->string pool-length #t))))))
+         (qcolumn "Pool Length" fn fn #:default-visible? #t))
```

Comment:
Can you please change `#:default-visible` to `#f`.  This is intended for the fields which should be visible by default when a new user installs the application for the first time, and I want to keep that list of fields short.  It will not affect your ability to add or remove the field from the column view using the "View / Setup Columns" menu option.  Other than that, it looks good.

Reply:
Done